perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -23,14 +24,30 @@
 	 * @param service Service JSON object
 	 * @return Service Created Service
 	 */
+	/*#
+	 * Creates a new service.
```

Comment:
Service object has more properties than name and description, you are free to create new constructor for Service object or use setters. At least script param should be taken on input since it doesn't have default value. You can note in a javadoc, that it should be constructed like ``./service_name`` (where anything else than ``[a-z,A-Z]`` is converted to undrescore ``_``.